.github/workflows/build.yaml

```diff
@@ -30,3 +30,14 @@ jobs:
     - name: Test
       working-directory: claat
       run: go test -v ./...
+
+
+  site:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: NPM Install
+        working-directory: site
+        run: npm install
```

Comment:
Actually, coming to think of it, given #884 problem, this shouldn't just `npm install` but also test ... what, `./node_modules/.bin/gulp dist` - probably?